tests/status/ignore.c

```diff
@@ -1077,3 +1077,81 @@ void test_status_ignore__negate_starstar(void)
     cl_git_pass(git_ignore_path_is_ignored(&ignored, g_repo, "code/projects/foo/bar/packages/repositories.config"));
     cl_assert_equal_i(0, ignored);
 }
+
+void test_status_ignore__ignore_all_toplevel_dirs_include_files(void)
+{
+	static const char *test_files[] = {
+		"empty_standard_repo/README.md",
+		"empty_standard_repo/src/main.c",
+		"empty_standard_repo/src/foo/foo.c",
+		"empty_standard_repo/dist/foo.o",
+		"empty_standard_repo/dist/main.o",
+		NULL
+	};
+
+	make_test_data("empty_standard_repo", test_files);
+	cl_git_mkfile(
+		"empty_standard_repo/.gitignore",
+		"/*/\n"
+		"!/src\n");
```

Comment:
Good call. I did a survey of the other test functions, and most of the have the first argument on the next line, and tests that use `make_test_data` tend to not have the space between it and `cl_git_mkfile`, so I'll push a new commit standardizing on that form.